src/NServiceBus.Transport.RabbitMQ/Administration/BrokerVerifier.cs

```diff
@@ -39,6 +32,13 @@ public Version BrokerVersion
 
     public async Task Initialize(CancellationToken cancellationToken = default)
     {
+        var all = BrokerRequirementChecks.Version310OrNewer | BrokerRequirementChecks.StreamsEnabled;
+
+        if (disabledBrokerRequirementChecks == all && !validateDeliveryLimits)
+        {
+            return;
```

Comment:
It looks like in the case of disabling everything we are no longer logging a warning?

Reply:
That happens in `VerifyRequirements`.